From 818ab22db9b61723fe26bfb8e4b958984ab0f838 Mon Sep 17 00:00:00 2001
From: Jack Olivieri <jolivieri@anaconda.com>
Date: Mon, 6 Jan 2025 21:12:35 +0100
Subject: [PATCH] fix install scripts

---
 source/CMakeLists.txt | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/source/CMakeLists.txt b/source/CMakeLists.txt
index 1d1a613aa..b819d0f32 100644
--- a/source/CMakeLists.txt
+++ b/source/CMakeLists.txt
@@ -582,6 +582,7 @@ endif()
 if(SVTHEVC_FOUND)
     target_link_libraries(x265-static ${SVT_HEVC_LIBRARY})
 endif()
+
 install(TARGETS x265-static
     LIBRARY DESTINATION ${LIB_INSTALL_DIR}
     ARCHIVE DESTINATION ${LIB_INSTALL_DIR})
@@ -612,6 +613,7 @@ if((WIN32 AND ENABLE_CLI) OR (WIN32 AND ENABLE_SHARED))
         install(FILES "${PROJECT_BINARY_DIR}/libx265.pdb" DESTINATION ${BIN_INSTALL_DIR} CONFIGURATIONS RelWithDebInfo OPTIONAL NAMELINK_ONLY)
     endif()
 endif()
+
 if(CMAKE_RC_COMPILER)
     # The resource compiler does not need CFLAGS or macro defines. It
     # often breaks them
@@ -789,3 +791,16 @@ if(hasParent)
         set(PLATFORM_LIBS ${PLATFORM_LIBS} PARENT_SCOPE)
     endif(PLATFORM_LIBS)
 endif(hasParent)
+
+if(ENABLE_HDR10_PLUS)
+    install(TARGETS hdr10plus-shared
+        LIBRARY DESTINATION ${LIB_INSTALL_DIR}
+        ARCHIVE DESTINATION ${LIB_INSTALL_DIR})
+endif()
+install(TARGETS x265-shared
+        LIBRARY DESTINATION ${LIB_INSTALL_DIR}
+        ARCHIVE DESTINATION ${LIB_INSTALL_DIR})
+# Produce a pkg-config file
+configure_file("x265.pc.in" "x265.pc" @ONLY)
+install(FILES       "${CMAKE_CURRENT_BINARY_DIR}/x265.pc"
+        DESTINATION "${LIB_INSTALL_DIR}/pkgconfig")
-- 
2.39.5 (Apple Git-154)

