diff --git a/source/common/CMakeLists.txt b/source/common/CMakeLists.txt
index 12b643ad5..83fc07ec5 100644
--- a/source/common/CMakeLists.txt
+++ b/source/common/CMakeLists.txt
@@ -131,9 +131,11 @@ endif()
 string(REPLACE ";" " " VERSION_FLAGS "${VFLAGS}")
 set_source_files_properties(version.cpp PROPERTIES COMPILE_FLAGS ${VERSION_FLAGS})
 
-check_symbol_exists(strtok_r "string.h" HAVE_STRTOK_R)
-if(HAVE_STRTOK_R)
-    set_source_files_properties(param.cpp PROPERTIES COMPILE_FLAGS -DHAVE_STRTOK_R=1)
+if(NOT WIN)
+    check_symbol_exists(strtok_r "string.h" HAVE_STRTOK_R)
+    if(HAVE_STRTOK_R)
+        set_source_files_properties(param.cpp PROPERTIES COMPILE_FLAGS -DHAVE_STRTOK_R=1)
+    endif()
 endif()
 
 if(GCC AND CC_HAS_NO_NARROWING)
