diff --git a/source/CMakeLists.txt b/source/CMakeLists.txt
index 2b26f3cf1..e525fdc3a 100755
--- a/source/CMakeLists.txt
+++ b/source/CMakeLists.txt
@@ -637,6 +637,10 @@ if(ENABLE_SHARED)
                     ${X265_RC_FILE} $<TARGET_OBJECTS:encoder> $<TARGET_OBJECTS:common> $<TARGET_OBJECTS:dynamicHDR10>)
         add_library(hdr10plus-shared SHARED $<TARGET_OBJECTS:dynamicHDR10>)
 
+        if(APPLE)
+            set_target_properties(hdr10plus-shared PROPERTIES MACOSX_RPATH 1)
+        endif()
+
         if(MSVC)
             set_target_properties(hdr10plus-shared PROPERTIES OUTPUT_NAME libhdr10plus)
         else()
